qiskit/utils/quantum_instance.py

```diff
@@ -367,24 +378,44 @@ def __str__(self) -> str:
 
         return info
 
-    def transpile(self, circuits):
+    def transpile(self, circuits, stage=TranspileStage.ALL):
```

Comment:
```suggestion
    def transpile(self, circuits, stage=TranspileStage.ALL) -> None:
```
And might check other places.

Reply:
the whole quantum instance could use some type hints, but I don't want to touch more than necessary in this PR 😄

Reply:
Well it did have typehints in Aqua but the migrated version here I think was changed in places so they are now part of the docstring args as I think there were issues with having them as typehints around importing/docs or both.